src/common/api/swagger.yaml

```diff
@@ -6242,10 +6243,10 @@ definitions:
       - 'networks-same-address-families'
       - 'network-prefix-valid'
       - 'machine-cidr-equals-to-calculated-cidr'
-      - 'api-vip-defined'
-      - 'api-vip-valid'
-      - 'ingress-vip-defined'
-      - 'ingress-vip-valid'
+      - 'api-vips-defined'
```

Comment:
Last thing I knew was that the API changes for Dual Stack VIPs wouldn't be released on next version.
Maybe we could update the types to have both IDs, with a TODO to remove the current one, but then we're safe as to when the API change happens.